### DISCUSSION TITLE
Available for android?

### DISCUSSION THREAD

**Discussion by max5354** [Arlodotexe/strix-music#171] [General]
Is this app available for android? Or will you release this app for android? Abd yes than how much it takes time to release this app on android?

**Comment by Arlodotexe** [comment]
Yes, eventually.

At the moment, our primary focus is on stabilizing and finalizing our SDK, and our secondary focus is on implementing the Zune Desktop shell.

The Windows app should work as is, out of the box. The Android and Web versions will get the attention they need when work on the SDK slows down a bit more.

The app has the potential to be released on Windows (UWP or WPF), Android, Web, iOS, macOS, and Linux. We use [Uno Platform](https://platform.uno/) to share 99% of our codebase x-plat, and any contribution made to Uno brings us a bit further towards supporting non-Windows platforms.

I'll update this discussion when we begin focusing on the Android app. Otherwise, anyone is free to contribute if they want to see the Android version come to life sooner than later.
